build/templates/backend/scripts/arbitrum_archive.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+{{define "main" -}}
+
+set -e
+
+INSTALL_DIR={{.Env.BackendInstallPath}}/{{.Coin.Alias}}
+DATA_DIR={{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend
+
+NITRO_BIN=$INSTALL_DIR/nitro
+
+$NITRO_BIN \
+  --chain.name arb1 \
+  --init.url https://snapshot.arbitrum.foundation/arb1/nitro-archive.tar \
+  --init.download-path $DATA_DIR/tmp \
+  --auth.jwtsecret $DATA_DIR/jwtsecret \
+  --persistent.chain $DATA_DIR \
+  --parent-chain.connection.url http://127.0.0.1:8136 \
```

Comment:
For the `archive` version, port `8116` should be needed instead of `8136`

Reply:
https://github.com/trezor/blockbook/pull/1112/commits/b69fca0f86acde2a5f275ab4f2501a2d02a2d6b1